firefox_ui_tests/functional/security/test_no_certificate.py

```diff
@@ -49,8 +49,7 @@ def test_no_certificate(self):
         Wait(self.marionette).until(lambda _: self.identity_popup.is_open)
 
         # Check the idenity popup doorhanger
-        self.assertEqual(self.identity_popup.element.get_attribute('className'),
-                         'unknownIdentity', 'The Larry UI is unknown (aka Grey)')
+        self.assertEqual(self.identity_popup.element.get_attribute('connection'), 'not-secure')
```

Comment:
It's totally the same check here. The `className` simply doesn't exist anymore, and is covered via `connection` now.